pkg/ccl/changefeedccl/metrics.go

```diff
@@ -88,6 +89,10 @@ type AggMetrics struct {
 		syncutil.Mutex
 		sliMetrics map[string]*sliMetrics
 	}
+
+	// TODO(baptist): This doesn't really belong here, but is easier than
+	// threading the NetMetrics through all the other places.
+	NetMetrics *cidr.NetMetrics
```

Comment:
agree with this comment. can you make a gh issue to clean this up and reference it in this comment pls

Reply:
Filed #130358 to track this.